app/modelmodule/logging.py

```diff
@@ -0,0 +1,36 @@
+"""Entities for logging permissions changes and other such things."""
```

Comment:
GGive has been using SimpleHistory to track model changes, it was a super light-weight integration and does just the trick (adding it to the middleware too):
from simple_history.models import HistoricalRecords
class AdminPermission(db.Model):
  ...
  history = HistoricalRecords()

Reply:
I don't think we can use that since we don't use Django's ORM, but it does seem worth considering other more generalized options, since we have other stuff we want to log in a similar way (API key changes are already logged much like what I have here, and eventually I want to do the same thing for repo settings and the global config). I'm going to drop this file from this PR and just merge `admin_acls.py` in for now, and do the logging in a separate PR later.